src/MongoDB/CursorId.c

```diff
@@ -88,33 +91,32 @@ static phongo_create_object_retval php_phongo_cursorid_create_object(zend_class_
 #else
 	{
 		zend_object_value retval;
-		retval.handle = zend_objects_store_put(intern, (zend_objects_store_dtor_t) zend_objects_destroy_object, php_phongo_cursorid_free_object, NULL TSRMLS_CC);
+		retval.handle   = zend_objects_store_put(intern, (zend_objects_store_dtor_t)zend_objects_destroy_object, php_phongo_cursorid_free_object, NULL TSRMLS_CC);
 		retval.handlers = &php_phongo_handler_cursorid;
 
 		return retval;
 	}
 #endif
 } /* }}} */
 
-static HashTable *php_phongo_cursorid_get_debug_info(zval *object, int *is_temp TSRMLS_DC) /* {{{ */
+static HashTable* php_phongo_cursorid_get_debug_info(zval* object, int* is_temp TSRMLS_DC) /* {{{ */
 {
-	php_phongo_cursorid_t  *intern;
+	php_phongo_cursorid_t* intern;
 #if PHP_VERSION_ID >= 70000
-	zval                    retval;
+	zval retval;
 #else
-	zval                    retval = zval_used_for_init;
+	zval retval = zval_used_for_init;
```

Comment:
Should we macroify this too? Just like I suggested for ``zval*`` vs ``zval`` for the return value?

Reply:
What would you use for PHP 7, `= { 0 }`?